docs/basics/publish-an-extension.md

```diff
@@ -16,10 +16,14 @@ Open your terminal, navigate to your extension directory, and run `npm run build
 
 ### Publish your extension
 
-To share your extension with others, navigate to your extension directory, and run `npm run publish` to publish your extension. You will be asked to authenticate with GitHub because the script will automatically open a pull request in our [repository](https://github.com/raycast/extensions).
+To share your extension with others, navigate to your extension directory, and run `npm run publish` to publish your extension. (If that command is not available, run `npx @raycast/api@latest publish` instead.)
```

Comment:
```suggestion
To share your extension with others, navigate to your extension directory, and run `npm run publish` to publish your extension. 

{% hint style="info" %}

It is possible that the `publish` script doesn't exist (usually because the extension was created before the template was updated to include it). In that case, you can add the following line in the `scripts` object of the package.json `"publish": "npx @raycast/api@latest publish"` and then run `npm run publish` again.

{% endhint %}
```